File: benches/mutations/variants/kotlin/seed1.t3-loop-swap.kt
fun summarizeScores(scores: List<Int>, threshold: Int): List<String> {
    val summaries = mutableListOf<String>()
    var runningTotal = 0
    for (i in scores.indices) {
        val score = scores[i]
        if (score < threshold) {
            continue
        }
        runningTotal += score
        var grade = "poor"
        if (score >= 90) {
            grade = "excellent"
        } else if (score >= 50) {
            grade = "adequate"
        }
        summaries.add(grade)
    }
    if (runningTotal > 250) {
        summaries.add("aggregate")
    }
    return summaries
}

File: benches/mutations/variants/typescript/seed1.t3-loop-swap.ts
function summarizeScores(scores: number[], threshold: number): string[] {
  const summaries: string[] = [];
  let runningTotal = 0;
  for (let i = 0; i < scores.length; i++) {
    const score = scores[i];
    if (score < threshold) {
      continue;
    }
    runningTotal += score;
    let grade = "poor";
    if (score >= 90) {
      grade = "excellent";
    } else if (score >= 50) {
      grade = "adequate";
    }
    summaries.push(grade);
  }
  if (runningTotal > 250) {
    summaries.push("aggregate");
  }
  return summaries;
}
